static/js/sefaria/textCache.js

```diff
@@ -0,0 +1,75 @@
+class SegmentVersionsFofLanguage {
```

Comment:
typo, should be SegmentVersionsForLanguage but perhaps even better would be"
`SegmentLanguageVersions`